Research/Research/RSDFormDataType.swift

```diff
@@ -92,8 +96,11 @@ public enum RSDFormDataType {
         /// to an `RSDDateRange` or `RSDNumberRange` to box the allowed values.
         case year
         
+        /// A `Codable` object. This is an object that can be represented using a JSON or XML dictionary.
```

Comment:
I didn't think XML was directly representable as a dictionary...? I mean, if there's a built-in XML decoder and that's what it outputs, then ok

Reply:
well, for Swift, there is. that's basically what a plist is.